pkg/cli/admin/upgrade/upgrade.go

```diff
@@ -460,3 +452,23 @@ func checkForUpgrade(cv *configv1.ClusterVersion) error {
 	}
 	return nil
 }
+
+func findUpdateFromConfigVersion(config *configv1.ClusterVersion, version string) *configv1.Update {
+	for _, update := range config.Status.AvailableUpdates {
+		if update.Version == version && len(update.Image) > 0 {
+			return &configv1.Update{
+				Version: version,
+				Image:   update.Image,
+			}
+		}
+	}
+	for _, history := range config.Status.History {
```

Comment:
Is it okay to bypass the channel lookup for "available updates", i.e. is it okay to assume that just because a version was in the channel at some point in "history" that it still is? For example, the cluster may have been updated to 4.6.0-fc.5 in the past but we have since pulled that version for some reason.

Reply:
Let's collapse warning about reaching into history into [this thread][1].

[1]: https://github.com/openshift/oc/pull/566#discussion_r488575538